### ISSUE TITLE
Add NFT Factory 

### ISSUE THREAD

**Issue by kerrilija** [asastats/channel#332]
A new NFT mint/buy/sell project called [NFT Factory](https://nftfactory.org/) by  [Headline Inc.](http://www.headline-inc.com) is live. Keep in mind that it is not fully developed yet.

**Comment by ipaleka** [comment]
This is so early in development that we should avoid implementing it right now. I spent two hours yesterday at the very beginning, purchasing some NFTs as an example. I have managed to successfully buy one of those and at the same time, I created dozens of opt-in transactions as the first step of failed purchases.

Also, I created two Algorand accounts for which I got a private key (passphrase) when I wanted to buy some NFTs.

**Comment by kerrilija** [comment]
Definitely. That's the reason I've mentioned it's not fully developed yet.

**Comment by kerrilija** [comment]
Quite a bit of NFTs appeared on the platform. Still feels kinda buggy tho.
